gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -242,6 +243,24 @@ def GetItemDescription (item, itemtype):
 	if (itemtype & 2):
 		text = item["ItemDesc"]
 
+	if GameCheck.IsIWD2():
+		usabilityMask = item["UsabilityBitmask"]
+
+		# restrictions apply, so we list them
+		if usabilityMask > 0:
+			text = GemRB.GetString(text) + "\n\n"
+			text += "[color=ffff00ff]" + GemRB.GetString(31530) + "[/color]\n  "
+
+			if usabilityMask & 0xE000: # alignment
+				text += GetIWD2RestrictedAlignments(usabilityMask)
+			if usabilityMask & 0x31000: # stance
+				text += GetIWD2RestrictedStances(usabilityMask)
+			if usabilityMask & 0x3F800000: # race
+				text += GetIWD2RestrictedRaces(usabilityMask)
+			if usabilityMask & 0x7FF: # classes
```

Comment:
I suggest, just for future proofing, instead of 0x7ff: 0xfff to give space for another class like the shaman.

But isn't the check too weak, since it covers only the class range? 00bard02 has no class bits set, only kit ones.

Reply:
Sorry, scrap that last part, I was looking at the wrong field.

Reply:
> I suggest, just for future proofing, instead of 0x7ff: 0xfff to give space for another class like the shaman.

Can't say much about this. It's _Unknown_ in IESDP and since IWD2 has no EE, I guess there won't be one any soon.

> But isn't the check too weak, since it covers only the class range? 00bard02 has no class bits set, only kit ones.

This item has class bits set for me, all but the bard class, and all kit ones, so this should fit.

Reply:
We already support the shaman and a 13th class in core, so this would makes it implementable through mods without touching our code.

Reply:
Alright